packages/frontend/src/pages/insight-page/insight-page.tsx

```diff
@@ -154,14 +153,7 @@ export const InsightPage = ({ isExport = false }) => {
 
       {insight && (
         <Routes>
-          <Route
-            path="edit/*"
-            element={
-              <SecureRoute>
-                <InsightDraftSwitcher insight={insight} onRefresh={refreshInsight} />
-              </SecureRoute>
-            }
-          />
+          <Route path="edit/*" element={<InsightDraftSwitcher insight={insight} onRefresh={refreshInsight} />} />
```

Comment:
Is the SecureRoute removed here because the InsightPage should already be within a SecureRoute?

Reply:
Yep exactly